modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/data_set_display/_data_set_display.scss

```diff
@@ -162,6 +143,10 @@
 			margin-bottom: map-get($spacers, 3);
 		}
 
+		.management-bar {
```

Comment:
Since this hasn't been merged yet, can we remove this and add `c-mb-0` to `<ClayManagementToolbar className="justify-content-space-between">`? We're already using utility classes there we can bundle this with that.

Reply:
I don't see why not, @diegonvs, we can do it in a followup if we want to push this one since it's already green, @brunobasto, @diegonvs